WHITEPAPER.md

```diff
@@ -213,11 +213,11 @@ At this point the broadcaster can begin streaming video segments towards the tra
 4. **Transcoder** -> **Broadcaster**: send output streamID and receipt that the job is accepted.
 5. **Broadcaster** -> **Transcoder**: send stream segments, which contain signatures verifying the input data.
 7. **Transcoder** performs transcoding and makes new output stream available on network
-9. **Transcoder**: Store a transcode claim for each segment of transcoding work. A transcode claim has the following fields.
+9. **Transcoder**: Store a transcode receipt for each segment of transcoding work. A transcode receipt has the following fields.
 
-| Transcode Claim Field | Description |
+| Transcode Receipt Field | Description |
 |-------|------------|
-| **JobID** | Identifies the origin node and stream that this segment belongs to. | 
+| **StreamID** | Identifies the origin node and stream that this segment belongs to. | 
 | **Sequence Number** | The sequential order that this segment belongs in the original stream. |
 | **Input Data hash** | The hash of the input segment data payload. |
 | **Transcoded Data hash** | The hash of the output data after transcoding this segment. |
```

Comment:
We talked about this offline yesterday - currently we assume a transcoder is responsible for transcoding a single input segment into all the necessary renditions specified in the transcodingOptions parameter (i.e. 5 renditions -> 5 transcoded segments). As a result of that the transcoded data hash is actually the hash of the concatenation of the hashes of each transcoded segment. Might not be necessary to specify here though since we might change this in the future?